requirements.txt

```diff
@@ -1,5 +1,4 @@
 # Django/Framework Packages
-django>=1.8,<1.9
```

Comment:
@efischer19 If it supports up to 1.11, I would expect to see `django>=1.8,<2.0`, it shouldn't hurt anything to have the extra support there and will probably save some trouble down the line.